NitroxModel/Packets/VehicleMovement.cs

```diff
@@ -12,13 +11,12 @@ public class VehicleMovement : Movement
         public VehicleMovement(ushort playerId, VehicleMovementData vehicleMovementData) : base(playerId, vehicleMovementData.Position, vehicleMovementData.Velocity, vehicleMovementData.Rotation, vehicleMovementData.Rotation)
         {
             VehicleMovementData = vehicleMovementData;
-            DeliveryMethod = NitroxDeliveryMethod.DeliveryMethod.UNRELIABLE_SEQUENCED;
```

Comment:
Is this intended ?

Reply:
Yes, `DeliveryMethod` will be set in `Movement.cs` (the base class).